product_docs/docs/pgd/5/durability/comparing-commit-scopes.mdx

```diff
@@ -0,0 +1,82 @@
+
+This also gives resiliency to failure of N nodes, where N is the number of nodes required to acknowledge the transaction. 
+Commit scope syntax allows you to specify N as part of the rule.
+
+### How Synchronous Commit works
+
+A Synchronous Commit first commits to the local disk and then waits till the transaction is replicated to N other nodes, as per rules of the commit scope. 
+On receiving confirmation that the transaction has been received, applied or made durable, again based on commit scope settings, the client gets a completion acknowledgement of the transaction.
+
+Synchronous commit increases resiliency - it ensures that the transaction is received by more than N nodes before it is acknowledged to caller. 
+This makes it resilient to failure of N nodes.
+When conflicts occur, with Synchronous Commit the conflict resolution is always the default async PGD conflict resolution.
+
+### Network partitions and node crashes
+
+Consider a network partition where the originating node is on one side of the partition and the replicas on the other.
+We will look at the state of each kind, before and after connectivity is restored.
+
+#### Group Commit and network partitions
+
+With Group Commit and before restoration, the origin node will have prepared a transaction and wait for confirmations which will not arrive.
+It could timeout and abort the transaction or just wait if the timeout setting is long enough.
+Writes that are recieved by the replicas will be held back if the nodes have prepared transactions from the now-disconnected origin. If there are no prepared transactions, they can continue to recieve writes.
+
+When connectivity is restored, eventually where the origin node aborted the transaction, the prepared transaction on some or all of the replicas will follow in aborting the transaction.
+If there are prepared transactions on all nodes, including the origin, the reconciliation process will commit them. Until that happens, each prepared transaction will be holding locks that will prevent nodes that have resumed writing from overwriting the transaction's writes from other nodes 
+
+#### Synchronous Commit and network partitions
+
+With Synchronous Commit and before restoration, the origin node will have committed the transaction and be waiting for confirmations which won't arrive. 
+The client may cancel it or it could timeout, but the transaction stays committed on this node. 
+On the other side of the partition, it's possible that one of the replicas becomes write leader and that segment of the cluster starts writing to the other nodes.
+
+As the network restores, and if the commit hasn't reached the other nodes, it is possible that the writes on the non-origin side could conflict with the writes from the origin node. The default async conflict resolution may choose one over the other and give errors or potential inconsistencies.
```

Comment:
As the network restores, and the commit from the separated origin arrives, it could conflict with the writes from the other nodes and the async conflict resolution may choose one over the other.